packages/headless/src/app/reducers.ts

```diff
@@ -57,6 +58,7 @@ export const searchHub = searchHubReducer;
 export const debug = debugReducer;
 export const resultPreview = resultPreviewReducer;
 export const version = versionReducer;
+export const trigger = triggerReducer;
```

Comment:
Should we use the plural `triggers` if the reducer is going to support all 4 triggers?